### PR TITLE
+107B: make the sound better + sync to animation

### PR DESCRIPTION
**Pull request by rmmh**
[demo](https://rawgit.com/rmmh/FastestWebsiteEver/music/client/index.html)

### CODE REVIEW

**Comment by diracdeltas** [comment]
wow my rabbit definitely likes this version better

File: Makefile [hunk 1]
```diff
@@ -1,7 +1,8 @@
 all: server/index.html server/cpkthttp
 
 server/index.html: client/index.html
-	./utils/deflate.py client/index.html server/index.html
+	sed 's:^\s*\|\s*$\|/\*.*\*/::g' client/index.html | tr -d '\n' > server/index.html.stripped
+	zopfli --i1000 -c --deflate server/index.html.stripped > server/index.html
```
Comment (diracdeltas) [inline Makefile:5]: @evq will this run ok on the server?

Comment (evq) [reply Makefile:5]: not atm, but that's easily fixed

Comment (evq) [reply Makefile:5]: @diracdeltas this should be good to go server side now (zopfli installed)

**Comment by alice-sawatzky** [comment]
incidentally, this fixes #14

**Pull request closed without merging**